Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -50,8 +51,15 @@ public override SceneQueryType SceneQueryType
         /// <remarks>
         /// Only [NearInteractionGrabbables](xref:Microsoft.MixedReality.Toolkit.Input.NearInteractionGrabbable) in one of the LayerMasks will raise events.
         /// </remarks>
+        [Obsolete("Use PrioritizedLayerMasksOverride instead")]
```

Comment:
It's a little unintuitive that we would be deprecating the grab mask but not the poke mask, I would expect that we would deprecate these thing together, since as a dev I would wonder "why grab masks deprecated but not poke masks?". Additionally, if we deprecate this, we will need to mention it in the release notes. I'm not sure deprecating the field right now is needed to fix this bug, so how about we consider this for a future PR instead? I'm a little worried that pulling the string on also deprecating the poke layer masks could cause more complexity than we expect. Thoughts?

Reply:
I think we can push back the deprecation for a later more dedicated re-write, I'll follow through with Kurtis's OnPreSceneQuery suggestion for now and make an item to rewrite the PioritizedLayerMask stuff